src/custom/ecospheres/views/HomeView.vue

```diff
@@ -56,6 +59,54 @@ useHead({
         </div>
       </div>
     </section>
+    <section class="fr-container">
+      <ul class="fr-grid-row fr-grid-row--gutters fr-py-12v" role="list">
```

Comment:
D'après la maquette on a un padding de 24px ici (et 24 sur le bloc au dessus aussi).
Soit il faut mettre du margin à la place sur la section plutôt que la liste pour que les 2 puissent collapse soit il faut réduire le padding `fr-py-6v` je pense.
Mais vu que le bloc au dessus est dans une autre PR je sais pas comment tu veux traiter ça in fine.
Désolé je l'ai pas vu précédemment.

![image](https://github.com/user-attachments/assets/ce2381cf-c2e5-4db8-a86c-42f93c77b1c8)

Reply:
J'aimerai régler ça dans la pr https://github.com/opendatateam/udata-front-kit/pull/556 du titre + moteur si t'es ok @narduin ?

Reply:
Yes pas de soucis